flashinfer/jit/cubin_loader.py

```diff
@@ -133,26 +138,24 @@ def load_cubin(cubin_path, sha256) -> bytes:
     return b""
 
 
-def get_cubin(name, sha256, file_extension=".cubin", session=None):
+def get_cubin(file_name: str, sha256: str, session=None) -> bytes:
```

Comment:
please also update the c++ side, where calling `getCubin` defined in `include/flashinfer/cubin_loader.h`. They leverage the default value of file extension to append `.cubin` automatically.

Reply:
Thanks, I missed those usages. I assumed all cubin file paths would come from the Python side somehow.